Jenkinsfile

```diff
@@ -25,6 +25,7 @@ def config = jobConfig {
     timeoutHours = 4
     runMergeCheck = false
     downStreamValidate = true
+    downStreamRepos = ["common",]
```

Comment:
@ijuma this is intended. it will trigger a build of the common repo after this finishes. since the common repo is where the version range property for kafka is defined we need to trigger that build so it picks up the latest version of kafka that wast just built. Eventually we will automate the list of downstream repos so we don't need to manually define them.